media/css/firefox/mobile.scss

```diff
@@ -106,12 +119,43 @@ a.button.button-focus {
     }
 }
 
-html[dir="rtl"] .main-header {
-    @media #{$mq-desktop} {
-        text-align: right;
+@media #{$mq-desktop} {
+    .header-content {
+        display: table-row;
+    }
+
+    .header-title,
+    .header-info {
+        display: table-cell;
+        width: 50%;
+    }
+
+    .header-title {
+        padding-right: $header-product-tablet-padding / 2;
+        white-space: nowrap;
```

Comment:
I'm not sure this is going to work with localized text, as `nowrap` overrides the `width: 50%`. In the past, we've used a one-off `html[lang^="en-"]` selector to keep English copy from wrapping (in the event that's important here).

<img width="970" alt="screen shot 2018-02-05 at 5 06 19 pm" src="https://user-images.githubusercontent.com/317532/35833447-ec97bb44-0a96-11e8-9443-dc168d7d56c8.png">

Reply:
This is kind of the intended effect. It's a lot more like the comp I got from George than the default English display is:

![screen shot 2018-02-06 at 13 45 36](https://user-images.githubusercontent.com/854701/35885889-29c8d3c4-0b44-11e8-81d9-844e1d38b199.png)

But I agree this language and breakpoint look bad so I'll play around with it.

Reply:
My kingdom for container queries.

Reply:
Also, how did you get it to show the l10n fallbacks? My local dev shows the untranslated strings for things missing a translation.

Reply:
You can either set `Dev=False` in your `.env` file (the classy way), or just hack it to `if not l10n_has_tag('...')`.